cornflow-dags/DAG/call_cornflow.py

```diff
@@ -0,0 +1,63 @@
+# Full imports
```

Comment:
Ok, Can we store this script as an example on our drive, but not have it here for now?